documentation/src/test/java/org/hibernate/search/documentation/mapper/pojo/standalone/entrypoints/StandalonePojoEntryPointsIT.java

```diff
@@ -115,12 +115,12 @@ void searchSession_withOptions() {
 	@Test
 	void searchScope_fromSearchMapping() {
 		SearchMapping searchMapping = theSearchMapping;
-		// tag::searchScope-fromSearchMapping[]
```

Comment:
Hmm, since these tests aren't used in the docs, should we keep them, or move them to the standalone tests, if it's not yet covered there? 🙈

Reply:
I don't know... While we only show the ORM snippets in the documentation, we present them as working for both mappers... So it seems to make sense that we also test the Standalone POJO Mapper...

Also, it seems like to keep it all consistent we'd have to create similar tests for ORM, but then there's duplication... 

All that to say: I won't object to merging a PR with such changes, but I'm not convinced it would be better than the current state of things, so... not worth it?